frontend/src/features/collections/select-collection-start-page.ts

```diff
@@ -160,35 +188,22 @@ export class SelectCollectionStartPage extends BtrixElement {
             );
           }}
         >
-          ${when(
-            this.selectedSnapshot,
-            (snapshot) => html`
-              <btrix-badge
-                slot="suffix"
-                variant=${snapshot.status < 300 ? "success" : "danger"}
-                >${snapshot.status}</btrix-badge
-              >
-            `,
-          )}
-          ${when(this.selectedPage, (item) =>
-            item.snapshots.map(
-              ({ pageId, ts, status }) => html`
-                <sl-option value=${pageId}>
-                  ${this.localize.date(ts)}
-                  <btrix-badge
-                    slot="suffix"
-                    variant=${status < 300 ? "success" : "danger"}
-                    >${status}</btrix-badge
-                  >
```

Comment:
Could we display the badge (or maybe just an icon with a tooltip) for non-2xx status codes? While I like that we're getting rid of it, I do think it could still be useful to highlight pages that didn't load successfully

Reply:
They're being filtered out, do you mean keep them in and highlight as errors?